Project.toml

```diff
@@ -8,14 +8,15 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+TimerOutputs = "a759f4b9-e2f1-59dc-863e-4aeb61b1ea8f"
```

Comment:
I think we should move the TimerOutputs dependency to being a test-only dependency (so in the [extras] section below). The reason is it pulls in some other dependencies we don't need (Unicode, Crayons).

Reply:
That is fine, I am just putting it in for now since I am using it to test a performance regression (and it is easier to just include it in this branch so I can more easily switch back and forth with master).

Reply:
Ok I see - great!

Reply:
While debugging this pull request, I put a global timer in `ITensors.jl` called `GLOBAL_TIMER`, and some timings that you can turn on with `TimerOutputs.enable_debug_timings(ITensors)` in `dmrg`. What do you think of keeping it as a dependency to make it easier to keep timers like that in the code? If it is in `extras`, I guess we wouldn't be able to leave those timers in the code. I doubt anyone will really notice it is being installed, it is a very light dependency.